lib/client/keystore.go

```diff
@@ -190,37 +184,54 @@ func (fs *FSLocalKeyStore) DeleteKey(host string, username string) error {
 	return nil
 }
 
+// DeleteKeys removes all session keys from disk.
+func (fs *FSLocalKeyStore) DeleteKeys() error {
+	dirPath := filepath.Join(fs.KeyDir, sessionKeyDir)
+
+	err := os.RemoveAll(dirPath)
```

Comment:
what if you logout twice this will be an error? probably should handle not found error on the upper level

Reply:
Doesn't return an error if called twice: https://golang.org/pkg/os/#RemoveAll

```
RemoveAll removes path and any children it contains. It removes everything it can
but returns the first error it encounters. If the path does not exist, RemoveAll
returns nil (no error).
```